examples/cluster/multiple-filestorage/cluster.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: greptime.io/v1alpha1
```

Comment:
It's duplicated for `standalone-wal/`

Reply:
It better to delete this example ?

Reply:
already rename [dedicated-wal](https://github.com/GreptimeTeam/greptimedb-operator/pull/254/commits/702a5ea5be9a1aebc38396ec73ab6f01b5b27dd4) and [rename standalone-wal to dedicated-wal](https://github.com/GreptimeTeam/greptimedb-operator/pull/254/commits/76a8d67a62dbaced0581f22de70f4a6ade25292d)